src/Donations/index.tsx

```diff
@@ -160,6 +166,24 @@ function DonationInfo() {
         className="background-image"
         alt="Background image with trees"
       /> */}
+
+      {isMobile && (
+        <button
+          id={"backButtonSingleP"}
+          style={{
+            cursor: "pointer",
+            width: "fit-content",
+            position: "absolute",
+            zIndex: 1000,
+            background: "#fff",
+            padding: "7px",
+            borderRadius: "100%",
+          }}
+          onClick={goBack}
+        >
+          <BackButton color={"#000"} />
```

Comment:
There is a callback button for successful donations. I wonder which use case this pull requests solves. Can you describe what to do to reproduce this new code?

Reply:
The button you're referring to is displayed after the user has made a donation. 
This PR adds a back button in mobile view (<=767 px) which when clicked takes the user to callback_url parameter in the link  and is available in between the process of making a donation. 
To reproduce the result, try: https://donate-with-planet-rlvr6dqhs-planetapp.vercel.app/?callback_url=https%3A%2F%2Fplanet-webapp-2a398fy3i-planetapp.vercel.app%2Freforestemos-centro-sur